src/content/docs/fr/guides/endpoints.mdx

```diff
@@ -96,10 +96,12 @@ Tout ce qui est décrit dans la section sur les endpoints des fichiers statiques
 
 Mais, contrairement au mode `static`, lorsque vous configurez le mode `server`, les endpoints seront construits lorsqu'ils seront demandés. Cela débloque de nouvelles fonctionnalités qui ne sont pas disponibles au moment de la construction, et vous permet de construire des routes d'API qui écoutent les requêtes et exécutent du code de manière sécurisée sur le serveur au moment de l'exécution.
```

Comment:
It seems you missed the [change here](https://github.com/withastro/docs/pull/8721/files#diff-ec68536a5c805ae285402a236bf012980eb1ab123aa75cf00c64e8eb87fc36e1L97).
```suggestion
Mais, contrairement au mode `static`, lorsque vous activez le mode de rendu à la demande pour une route, le point de terminaison sera construit lorsqu'il sera demandé. Cela débloque de nouvelles fonctionnalités qui ne sont pas disponibles au moment de la construction, et vous permet de construire des routes d'API qui écoutent les requêtes et exécutent du code de manière sécurisée sur le serveur au moment de l'exécution.
```